ts/screens/authentication/LandingScreen.tsx

```diff
@@ -130,38 +146,26 @@ const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
   body: "authentication.landing.contextualHelpContent"
 };
 
-const styles = StyleSheet.create({
-  flex: {
-    flex: 1
-  },
-  uatCie: {
-    backgroundColor: IOColors.red
-  },
-  noCie: {
-    // don't use opacity since the button still have the active color when it is pressed
-    // TODO: Remove this half-disabled state.
-    // See also discussion on Slack: https://pagopaspa.slack.com/archives/C012L0U4NQL/p1657171504522639
-    backgroundColor: IOColors.noCieButton
-  },
-  fullOpacity: {
-    backgroundColor: variables.brandPrimary
-  },
-  link: {
-    textAlign: "center",
-    paddingBottom: 5,
-    paddingTop: 4.5,
-    lineHeight: 30
-  }
-});
-
 export const IdpCIE: SpidIdp = {
   id: "cie",
   name: "CIE",
   logo: "",
   profileUrl: ""
 };
 
+const BUTTON_SPACING = 24;
```

Comment:
I understand what are you suggesting, but in this case the scope of having `BUTTON_SPACING` here it's only to centralized the value `24` used many times. So that if there were the necessity to change the value `24` in, let's say `16`, we could do it just in one place.

About having an utility files where to store all standard sizes, I'm not sure right now if it should be a valid solution, because all the sizes we use in the DS components are of type `IOSpacer` so we cannot used different and not standard values. 

Moreover, the spacing between buttons are usually managed inside DS system "templates", like `FooterWithButtons` or `GradientScrollView` so, for now, I prefer to leave the spacing policy inside every screen when they deviate from templates and need a custom layout, like in this case.

Reply:
It's ok!